developer/deploying/platform.md

```diff
@@ -1,10 +1,10 @@
 # Reaction Platform
 
-Reaction Commerce offers a managed deployment option, the [Reaction Platform](http://getrxn.io/reaction-platform).
+Reaction Commerce offers a managed deployment option, [Reaction Platform](http://getrxn.io/reaction-platform).
 
-The `reaction-cli` incorporates functionality for any team to deploy Reaction Commerce to multiple environments. [Visit reactioncommerce.com](http://getrxn.io/reaction-platform) to get a demo or request an **invite token**.
+The `reaction-cli` incorporates functionality for any team to deploy Reaction Commerce to multiple environments. [Visit reactioncommerce.com](http://getrxn.io/reaction-platform) to get a demo.
 
-1.  [Request invite token](http://getrxn.io/reaction-platform).
+1.  Request invite token
```

Comment:
@sophiehe maybe we should add one more detail here about how to get an invite token in case people only read Step 1.

Reply:
or add another sentence like...`The following documentation is for paid customers of Reaction Platform`, just to make it extra clear!